docs/skopeo.1.md

```diff
@@ -0,0 +1,115 @@
+% SKOPEO(1) Skopeo Man Pages
+% Jhon Honce
+% August 2016
+# NAME
+skopeo -- Various operations with container images images and container image registries
+# SYNOPSIS
+**skopeo [gobal options] command [command options]**
```

Comment:
- s/gobal/global/
- `global options`, `command`, `command options` should be italic (replaceable), `[]` should not be bold (not literal), per **man-pages**(7).